lib/core/typeinfo.nim

```diff
@@ -67,12 +67,12 @@ type
   pbyteArray = ptr array[0.. 0xffff, int8]
 
   TGenericSeq {.importc.} = object
-    len, space: int
+    len, space, elemSize: int
```

Comment:
Still not acceptable to use 3 words here instead of 2. You can have the 'elemSize: int' in a 'when defined(gogc)' though.